news/2021-04-06-mappers-report-march.md

```diff
@@ -0,0 +1,173 @@
+
+Remember the [Labour of Love](/wiki/Contests/A_Labour_of_Love_Contest) contest? Submissions are in, and now it's on you to pick the winner, but you have only two votes to cast out of eight incredible mapsets. Don't forget that you also have only less than a week to vote!
+
+So make sure to check them all out and vote for your favorite. You can find all the submissions and download them on the [Labour of Love contest page](https://osu.ppy.sh/community/contests/115).
+
+We're excited to see who will take the trophy home!
+
+## <a id="around-community"></a>Around the Community
+
+Last but not least, we want to give other mapping-related projects a voice that does not fit in any of the categories above. All have the same goal: Helping the community. Should you be interested in it, make sure to check them out!
+
+### Community Mentorship Program Spring 2021
+
+The Mentorship program is back with another cycle! Are you interested in mapping but have no idea where to start? Or, are you looking for people to convey your ingenious mapping techniques? Then search no more. The [Community Mentorship Program](https://osu.ppy.sh/community/forums/topics/1273766 "Thread") is exactly what you are looking for.
+
+In this program, experienced mappers will teach people interested in how to create a beatmap. Whether it be osu!, osu!taiko, osu!catch or osu!mania, the program has mapping mentors from every game mode. While the mentor signups have already been closed, the mentee signups will open on **April 19th** and will remain up until **April 25th**. Save the date!
+
+If you have ever wanted to learn how to map, do not waste this chance and sign up now! You can find all the vital information in the forum thread linked above.
+
+### Advanced beatmap search is here!
+
+Always wanted to search for older beatmaps but hated having to scroll through the endless pages of the beatmap listing? Then we have good news for you! The [beatmap listing page](https://osu.ppy.sh/beatmapsets) has been significantly improved and now finally supports advanced searching functions. You can search for beatmaps ranked in a specific year, for example, by using `ranked=2010`. You may also search for specific keys for osu!mania beatmaps, via `keys=7`. 
+
+A complete list of possible search options can be found on the [Interface wiki page](/wiki/Interface#search).
+
+### dsco's Guide to Making and Ranking the First Mapset
+
+If you have ever wondered how a beatmap is created from scratch and carried all over to the ranked section, this is your lucky day because we have an answer for you! [dsco](https://osu.ppy.sh/users/4330511), a mapper and modder with quite a few years-long experience, has livestreamed for over 9 hours in order to guide his viewers into creating a beatmap and getting it ranked. Throughout his livestream, he began mapping [Litku Klemetti - Jaakuningatar (Cut Ver.)](https://osu.ppy.sh/beatmapsets/1421834) from scratch and carried it to the ranking queue at the very end. 
```

Comment:
```suggestion
If you have ever wondered how a beatmap is created from scratch and carried all over to the ranked section, you're in luck, because esteemed mapper and modder [dsco](https://osu.ppy.sh/users/4330511) has livestreamed for over 9 hours in order to guide his viewers through the entire process from start to finish with his set for [Litku Klemetti - Jaakuningatar (Cut Ver.)](https://osu.ppy.sh/beatmapsets/1421834).
```